pyproject.toml

```diff
@@ -50,6 +50,9 @@ pytest-xdist = "^2.1"
 sphinx = "^4.0"
 Werkzeug = "^2.0"
 
+[tool.poetry.group.dev.dependencies]
+pretend = "^1.0.9"
```

Comment:
There seems to be a group `dev-dependencies` already above. I guess those two should get merged.